### DISCUSSION TITLE
Requesting help in FeynCalc related evaluations

### DISCUSSION THREAD

**Discussion by leothestark** [FeynCalc/feyncalc#279] [General]
Issue:
I am using Mathematica 13 and FeynCalc 10. I wanted to use the Dirac matrices anticommutator trick to simplify the Dirac matrices. I looked up one of the forums where @vsht has replied regarding the same issue. The solution was to use "{DiracCanonical -> True}". I used that but I am not getting the expected results.

**Comment by vsht** [comment]
It's called `DiracOrder->True` is FeynCalc 10.
